GiniVision/Tests/CameraButtonsViewControllerTests.swift

```diff
@@ -279,4 +280,29 @@ final class CameraButtonsViewControllerTests: XCTestCase {
                        "the inner stack view should contain the captured images stack button")
     }
     
+    func testFlashStatusWhenOnByDefault() {
+        let giniConfiguration = GiniConfiguration()
+        
+        cameraButtonsViewController = CameraButtonsViewController(giniConfiguration: giniConfiguration,
+                                                                  isFlashSupported: true,
+                                                                  currentDevice: IphoneDevice())
+        
+        XCTAssertTrue(cameraButtonsViewController.flashToggleButton.isSelected,
+                      "the flash toggle should be selected when flash in on by default")
+        
+    }
+    
+    func testFlashStatusWhenOffByDefault() {
+        let giniConfiguration = GiniConfiguration()
+        giniConfiguration.flashOnByDefault = false
+        
+        cameraButtonsViewController = CameraButtonsViewController(giniConfiguration: giniConfiguration,
+                                                                  isFlashSupported: true,
+                                                                  currentDevice: IphoneDevice())
+        
+        XCTAssertFalse(cameraButtonsViewController.flashToggleButton.isSelected,
+                       "the flash toggle should be selected when flash in on by default")
```

Comment:
"Off" by default. And since it's an error caused by copy-pasting, maybe you should generalize the whole method.

Reply:
For only two test it's an overkill